examples/demo.rs

```diff
@@ -59,7 +59,7 @@ fn main() -> ! {
         clocks,
     );
 
-    let spi_interface = SPIInterface::new(spi, dc, cs);
+    let spi_interface = SPIInterface::new(spi, dc);
 
     // If you don't need the Chip Select pin, use this instead:
     // let spi_interface = SPIInterfaceNoCS::new(spi, dc);
```

Comment:
I think with the updated dependencies there is no way to pass the CS pin and you either need to manage it manually or use another mechanism like `embedded-hal-bus`. I've removed the comment to avoid confusion.